### PR TITLE
#8513: Add slack notifications to several more pipelines

### PR DESCRIPTION
**Pull request by dimitri-tenstorrent**
Issue #8513

### CODE REVIEW

File: .github/workflows/build-and-unit-tests.yaml [hunk 1]
```diff
@@ -42,3 +42,8 @@ jobs:
         run: |
           source ${{ github.workspace }}/python_env/bin/activate
           ./tests/scripts/run_tests.sh --tt-arch $ARCH_NAME --pipeline-type post_commit --dispatch-mode slow
+      - uses: ./.github/actions/slack-report
+        if: ${{ failure() }}
+        with:
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          owner: U0593J2RQRZ # Bill Teng
```
Comment (tt-rkim) [inline .github/workflows/build-and-unit-tests.yaml:49]: posix newline
Screenshot 2024-05-22 at 9 26 37 AM

File: .github/workflows/build-and-unit-tests.yaml [hunk 2]
```diff
@@ -42,3 +42,8 @@ jobs:
         run: |
           source ${{ github.workspace }}/python_env/bin/activate
           ./tests/scripts/run_tests.sh --tt-arch $ARCH_NAME --pipeline-type post_commit --dispatch-mode slow
+      - uses: ./.github/actions/slack-report
+        if: ${{ failure() }}
+        with:
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```
[the remark this answers is not in the document]
Comment (tt-rkim) [reply .github/workflows/build-and-unit-tests.yaml:48]: isn't that what we want for these?

[the remark this answers is not in the document]
Comment (dimitri-tenstorrent) [reply .github/workflows/build-and-unit-tests.yaml:48]: The action only activates for `main`. You can see the if statement within the .action file.

https://github.com/tenstorrent/tt-metal/pull/8685/files#diff-ac6cc283efb059f91f0272c7a9674fd479556c0ef750bae8301d8ce5cd671b10R13

**Review by TT-billteng — APPROVED** [review]
looks great!

**Review by tt-rkim — APPROVED** [review]

**Pull request merged**